.semaphore/sri_lanka_demo_deployment.yml

```diff
@@ -24,3 +24,7 @@ promotions:
     pipeline_file: sri_lanka_production_deployment.yml
     auto_promote:
       when: (branch = 'master' AND result = 'passed') OR (tag =~ '^release-.*' AND result = 'passed')
+  - name: Sri Lanka Production Deployment k8s
```

Comment:
Maybe we call this "Sri Lanka Production Deployment", and the other one we update to "Sri Lanka Production Deployment AWS" or "Old" or something